impl/src/test/java/org/ehcache/docs/Tiering.java

```diff
@@ -35,6 +51,18 @@
   public TemporaryFolder folder = new TemporaryFolder();
 
   @Test
+  public void tierSizing() {
+    // tag::heap[]
+    ResourcePoolsBuilder.newResourcePoolsBuilder().heap(10, EntryUnit.ENTRIES); // <1>
+    ResourcePoolsBuilder.newResourcePoolsBuilder().heap(10); // <2>
```

Comment:
No, that heap method is static and not an instance method on the builder. So let's indicate that. Otherwise its usage in a chain will effectively silently create a new builder. It should only ever be used as the first point and was actually a mistake to add it.